### DISCUSSION TITLE
OSP Community Conference: Pre-Registration and Announcement

### DISCUSSION THREAD

**Discussion by StephanSchaller** [Open-Systems-Pharmacology/Forum#1728] [Announcement]
Dear colleagues,

We are excited to announce the first Open Systems Pharmacology (PK-Sim and Mobi) community conference!

We are organizing a 1.5-day conference, where we will discuss topics related to PBPK modeling in general and, more specifically, our open-source software and its application. 

The meeting will be organized in Europe in September/October (likely in Germany or Switzerland).

In the spirit of an open community, we would like to hear from you about what you want to have covered in this event.

Please fill out the form below, where you can also indicate your attendance; we will keep you updated!

Further information on the registration for the event will follow.
 

[ OSP Symposium 2024: Topics & Format](https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.google.com%2Fforms%2Fd%2F1KgZ_dhlfeNOz7wIeusIAbxOOcNGOGyhhS64H4265Tv8%2Fviewform%3Fedit_requested%3Dtrue&data=05%7C02%7Cstephan.schaller%40esqlabs.com%7C77bb89a82cd74b4ab5ef08dc4144077b%7Cdedc8592564247f0a69e602e65c3d168%7C0%7C0%7C638457007648271267%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=0NjkoxQ5HPh4I3rG0bKrfbvhRp7qR%2FszDwlRTKWMaSA%3D&reserved=0)

On behalf of the OSP community meeting organization

**Comment by HenrikCordes** [comment]
We are very happy to announce that registration for the 1st OSP Community Conference this October is NOW OPEN !

Join us at the Novartis Campus in Basel from October 7-8 for an event filled with:
-talks about the latest applications and advancements in the OSP Suite
-reflections by regulators on submissions supported with (OSP-based) PBPK and QSP
-discussions on the sustainability, roadmap, and vision for the OSP community and its growth
-opportunities for collaboration within the community

More Info and a link to register can be found here: https://www.open-systems-pharmacology.org/conference.html
